p/scripts/main.js

```diff
@@ -937,15 +943,17 @@ function init_shortcuts() {
 					toggleContent(last, old_active, false);
 				}
 			}
-			return false;
+			ev.preventDefault();
+			return;
 		}
 
 		if (ev.altKey || ev.shiftKey) {
-			return true;
+			return;
```

Comment:
Using this approach, no difference when returning `true` of `false`?

Reply:
I can't find it on MDN at the moment, but `return false` in an old style event listener is equivalent to preventDefault(), stopPropagation() and possibly some other things like that as well.

The return value from addEventListener() is ignored.